instrumentor/controllers/agentenabled/pods_webhook.go

```diff
@@ -358,3 +386,50 @@ func getRuntimeInfoForContainerName(ic *odigosv1.InstrumentationConfig, containe
 	// if both are not found, return we don't have runtime info for this container
 	return nil
 }
+
+func CreateInitContainer(pod *corev1.Pod, dirsToCopy map[string]struct{}, config common.OdigosConfiguration) {
```

Comment:
This is a private function

```suggestion
func createInitContainer(pod *corev1.Pod, dirsToCopy map[string]struct{}, config common.OdigosConfiguration) {
```